src/app/ecosystem-explorer/project-form/components/EcosystemProjectForm.tsx

```diff
@@ -0,0 +1,225 @@
+'use client'
+
+import { zodResolver } from '@hookform/resolvers/zod'
+import { useForm } from 'react-hook-form'
+import { z } from 'zod'
+
+import type { CategoryMap } from '@/types/categoryTypes'
+
+import { Button } from '@/components/Button'
+import { ControlledForm } from '@/components/Form/ControlledForm'
+import { ControlledFormCheckbox } from '@/components/Form/ControlledFormCheckbox'
+import { ControlledFormFileInput } from '@/components/Form/ControlledFormFileInput'
+import { ControlledFormInput } from '@/components/Form/ControlledFormInput'
+import { ControlledFormListbox } from '@/components/Form/ControlledFormListbox'
+import { ControlledFormTextarea } from '@/components/Form/ControlledFormTextarea'
+import { FormField } from '@/components/Form/FormField'
+
+import {
+  BRIEF_CHARACTER_LIMIT,
+  MAX_FILE_SIZE_IN_BYTES,
+  NETWORK_USE_CASE_CHARACTER_LIMIT,
+} from '../constants'
+import { useSubmitEcosystemProjectForm } from '../hooks/useSubmitEcosystemProjectForm'
+import { EcosystemProjectFormSchema } from '../schema/form'
+import { getOptionsFromObject } from '../utils/getOptionsFromObject'
+import { getYearOptions } from '../utils/getYearOptions'
+
+import { FormSection } from './FormSection'
+
+type StringOrUndefined = string | undefined
+
+export type EcosystemProjectFormData = z.infer<
+  typeof EcosystemProjectFormSchema
+>
+
+type ProjectFormProps = {
+  categoryData: CategoryMap
+  subCategoryData: CategoryMap
+  initialValues: EcosystemProjectFormData
+}
+
+export function EcosystemProjectForm({
+  categoryData,
+  subCategoryData,
+  initialValues,
+}: ProjectFormProps) {
+  const yearOptions = getYearOptions('desc')
+  const categoryOptions = getOptionsFromObject(categoryData)
+  const subCategoryOptions = getOptionsFromObject(subCategoryData)
+
+  const form = useForm<EcosystemProjectFormData>({
+    resolver: zodResolver(EcosystemProjectFormSchema),
+    defaultValues: initialValues,
+  })
+  const isSubmitting = form.formState.isSubmitting
+
+  const submitForm = useSubmitEcosystemProjectForm()
+
+  return (
+    <ControlledForm<EcosystemProjectFormData>
+      form={form}
+      className="max-w-readable"
+      onSubmit={submitForm}
+    >
+      <FormSection
+        title="Personal Information"
+        description="We'll use this to verify your site updates and reach out to you in case of questions."
+      >
+        <ControlledFormInput<EcosystemProjectFormData>
+          name="name"
+          label="Name"
+          placeholder="Your Name"
+          disabled={isSubmitting}
+        />
+        <ControlledFormInput<EcosystemProjectFormData>
+          name="email"
+          label="Email"
+          placeholder="Your Email"
```

Comment:
This just says "Email" [in Figma](https://www.figma.com/design/0pBfQs5tKI6bx5ypPIfVRA/FIL.org-V4-Prototypes?node-id=5196-68950&t=bTzI5ZqPyweteofc-4), as a placeholder.